x/ecocredit/client/basket/tx.go

```diff
@@ -61,6 +61,14 @@ Flags:
 			curator explicitly acknowledges paying this fee and is not surprised to learn that the
 			paid a big fee and didn't know beforehand.
 		description: the description to be used in the basket coin's bank denom metadata.
+Example:
```

Comment:
there's an explicit `Example` field in the `Command` struct here - i used it in a couple of my CLI impl maybe we can do it here too?

Reply:
> i used it in a couple of my CLI impl maybe we can do it here too?

Sure, can you link me to one here.

Reply:
https://github.com/regen-network/regen-ledger/blob/release/v3.0.x/x/ecocredit/client/basket/query.go#L15-L20